src/AppDesktop.vue

```diff
@@ -38,6 +41,12 @@ export default {
     margin-left: auto;
     margin-right: auto;
   }
+
+  .connection-status {
+    position: absolute;
```

Comment:
```suggestion
    position: fixed;
```
This should fix improper rendering if page scrolled:
![Screenshot 2019-07-18 at 05 33 33](https://user-images.githubusercontent.com/9007851/61427210-99251200-a91d-11e9-9666-8982b06e1049.png)